examples/integration-tutorials/using-the-gmail-connector/README.md

```diff
@@ -44,15 +74,82 @@ gmail:GmailConfiguration gmailConfig = {
         }
     }
 };
+```
 
-gmail:Client gmailClient = new(gmailConfig);
+With the defined `gmailClient`, a function to send the mail needs to be defined. The sender and recipient of the email is specified and the client is used to send the message. 
+
+```ballerina
+
+function sendEmail(string email) returns http:Response {
+    string messageBody = email;
+    http:Response response = new;
+
+    string userId = "me";
+    gmail:MessageRequest messageRequest = {};
+    messageRequest.recipient = "someone@gmail.com";
```

Comment:
Can we separate the data (e.g.: email address) from the code? A separate properties file would be a possibility to enter user data. At least the data can be specified as constants at the beginning of the code.

Reply:
ok. I will add a separate properties file or constants

Reply:
Done